editor.planx.uk/src/pages/Team.tsx

```diff
@@ -311,15 +308,23 @@ const Team: React.FC = () => {
   );
   const [flows, setFlows] = useState<FlowSummary[] | null>(null);
 
-  const sortArray: SortableFields<FlowSummary>[] = [
-    { displayName: "Name", fieldName: "name" },
-    { displayName: "Last updated", fieldName: "updatedAt" },
+  const sortOptions: SortableFields<FlowSummary>[] = [
+    {
+      displayName: "Name",
+      fieldName: "name",
+      directionNames: { asc: "A - Z", desc: "Z - A" },
```

Comment:
Nice! When we have more use cases for this we _may_ find we want these display values centralised within the component, and just allow consumers to define by type (e.g. `direction: "alphabetical" | "date"`). This is perfect for now 👍